SmartDeviceLink/SDLLockScreenPresenter.m

```diff
@@ -168,9 +167,12 @@ - (void)sdl_dismissLockscreenWithCompletionHandler:(void (^ _Nullable)(void))com
     [[NSNotificationCenter defaultCenter] postNotificationName:SDLLockScreenManagerWillDismissLockScreenViewController object:nil];
 
     SDLLogD(@"Dismissing the lockscreen window");
+    _isDismissing = YES;
```

Comment:
When I test the Obj-C app on a iOS 12 device, the lockscreen can not be dismissed when:
1. Set the TDK driving status to "moving".
1. Connect the device to a SYNC 3.0 head unit using a USB cord. The lockscreen should show up.
1. Trigger a `DD_OFF` notification by setting the TDK driving status to "parked"

### Expected
The lock screen dismisses.

### Actual
The lock screen can not be dismissed.

Reply:
Fixed